compiler/rustc_feature/src/active.rs

```diff
@@ -679,6 +679,10 @@ declare_features! (
     /// Allows `let...else` statements.
     (active, let_else, "1.56.0", Some(87335), None),
 
+    /// Allows `#[must_not_suspend]`.
+    (active, must_not_suspend, "1.56.0", Some(83310), None),
+
```

Comment:
Note: 1.56.0 has already branched off as beta, so this will land in 1.57.0 or later.